local-cli/runIOS/__tests__/findMatchingSimulator-test.js

```diff
@@ -0,0 +1,195 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+'use strict';
+
+jest.dontMock('../findMatchingSimulator');
+
+const findMatchingSimulator = require('../findMatchingSimulator');
+
+describe('findMatchingSimulator', () => {
+  it('should find simulator', () => {
+    expect(findMatchingSimulator({
+        "devices": {
```

Comment:
@frantic Each one of these JSON blobs are unique to each test. I was thinking it'd be easier to read if each of them were inside the test. Thoughts?